cleverhans/future/tf2/attacks/fast_gradient_method.py

```diff
@@ -42,7 +47,7 @@ def fast_gradient_method(model_fn, x, eps, norm, clip_min=None, clip_max=None, y
     # Using model predictions as ground truth to avoid label leaking
     y = tf.argmax(model_fn(x), 1)
 
-  grad = compute_gradient(model_fn, x, y, targeted)
+  grad = compute_gradient(model_fn, loss_fn, x, y, targeted)
```

Comment:
Loss function should come as an optional parameter at the end to avoid breaking code calling compute_gradient directly

Reply:
In my opinion, `loss_fn` is essential argument for private function `compute_gradient`. and `loss_fn` is optional for public function `fast_gradient_method`. 

we have selected a default `loss_fn` at line 34-35 and put `loss_fn` to function compute_gradient.